### PR TITLE
Move ly lyc to stat page

### PR DESCRIPTION
**Pull request by leinacc**
Closes https://github.com/gbdev/pandocs/issues/439

I decided on putting these on top of STAT as they're mentioned in sentence 2 of STAT's description, so it's fresh info

Maybe title needs changing? Though LCD Status Registers feels wrong

### CODE REVIEW

File: src/STAT.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # LCD Status Register
 
+## FF44 — LY: LCD Y coordinate \[read-only\]
```
Comment (avivace) [inline src/STAT.md:3]: I think in any case they should be after the terminology box, as it's meant to be an introductory block or text to satisfy so me knowledge pre-requirements

File: src/STAT.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ A *dot* is the shortest period over which the PPU can output one pixel: is it eq
...
+
+LY indicates the current horizontal line, which might be about to be drawn,
+being drawn, or just been drawn. LY can hold any value from 0 to 153, with
+values from 144 to 153 indicating the VBlank period.
+
+## FF45 — LYC: LY compare
+
+The Game Boy permanently compares the value of the LYC and LY registers.
```
Comment (ISSOtm) [inline src/STAT.md:17]:
```suggestion
The Game Boy constantly compares the value of the LYC and LY registers.
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
I dunno, "LCD status registers" seems fine to me, since they are multiple registers that report the PPU's drawing status.

That said, the name in `SUMMARY.md` should also be changed, then.

**Comment by ISSOtm** [comment]
That last commit is indeed a good idea, but then I'm thinking we may want to actually introduce and link to a new section at the bottom to specifically the explanation of the LCD modes. What do you think?

**Comment by leinacc** [comment]
> That last commit is indeed a good idea, but then I'm thinking we may want to actually introduce and link to a new section at the bottom to specifically the explanation of the LCD modes. What do you think?

It sounds like a good idea just seeing those other pages link to STAT by mentioning modes.
I'll place the header above `The LCD controller operates on a 2^22 Hz = 4.194 MHz dot clock.` where it starts to describe them

**Review by avivace — APPROVED** [review]

**Review by ISSOtm — APPROVED** [review]

**Comment by ISSOtm** [comment]
Thank you!

**Pull request merged**